docs/docs-ref-autogen/excel/excel/excel.interfaces.linkedentitydatadomaincollectionloadoptions.yml

```diff
@@ -12,9 +12,6 @@ remarks: >-
   \[ [API set: ExcelApi
   1.19](/javascript/api/requirement-sets/excel/excel-api-requirement-sets) \]
 
-
-  This interface isn't supported in Excel on the web.
```

Comment:
These notes were not supposed to be removed. I'll re-add them in a DefinitelyTyped PR. This isn't merge-blocking for the ref docs, the notes can just get picked back up next week.